src/mono/wasm/runtime/startup.ts

```diff
@@ -28,6 +28,7 @@ import { instantiate_wasm_asset, mono_download_assets, resolve_asset_path, start
 import { BINDING, MONO } from "./net6-legacy/imports";
 import { readSymbolMapFile } from "./logging";
 import { mono_wasm_init_diagnostics } from "./diagnostics";
+import { preAllocatePThreadWorkerPool, instantiateWasmPThreadWorkerPool } from "./pthreads/browser";
```

Comment:
I think you checked this before, but please check again that non-MT build doesn't have the code included (in release build).

Reply:
Yep, I verified that this is gone